### PR TITLE
Spatial Computing Contribution 

### PR DESCRIPTION
**Pull request by ArafatAbdussalam**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,17 @@ Access a range of materials for personal growth, including goal setting, product
 - [**Advanced Level**](./personalDevelopment/personaldevelopment/#-advanced-level)
 - [**Books**](./personalDevelopment/personaldevelopment/#-books)
 
+
+## ᯅ Spatial Computing
```
Comment (Charity10) [inline README.md:62]: Should be added to thee table of content for easy navigation

**Comment by ArafatAbdussalam** [comment]
Hi @Charity10 The table of content link isn’t working from my end. I followed the same pattern of linking. Could you help me review what must have been done wrong.

File: README.md [hunk 2]
```diff
@@ -58,6 +59,16 @@ Access a range of materials for personal growth, including goal setting, product
 - [**Advanced Level**](./personalDevelopment/personaldevelopment/#-advanced-level)
 - [**Books**](./personalDevelopment/personaldevelopment/#-books)
 
+## ᯅ Spatial Computing
```
Comment (Charity10) [inline README.md:62]: remove boxed ? here

File: README.md [hunk 3]
```diff
@@ -9,7 +9,8 @@ Welcome to the **Learning Resources Repository**! This repository is designed to
 1. [Frontend Development](#-frontend-development)
 2. [Backend Development](#-backend-development)
 3. [Mobile Development](#mobile-development)
-3. [Personal Development](#-personal-development)
+4. [Personal Development](#-personal-development)
+5. [Spatial Computing](#-spatial-computing)
```
Comment (Charity10) [inline README.md:13]: remove the '-' here

File: README.md (hunk 2 above)
Comment (ArafatAbdussalam) [reply README.md:62]: Can you explain better. What’s boxed?

Comment (ArafatAbdussalam) [reply README.md]: It works now. I removed the emoji.

File: README.md (hunk 3 above)
Comment (ArafatAbdussalam) [reply README.md]: Thank you. I have done that.

**Comment by Charity10** [comment]
Hi Arafat, this branch has conflict
you need to pull again and update the readme file

A new resource has been added to no 5

**Comment by ArafatAbdussalam** [comment]
Hi Mercy I have resolved the conflict. Please check it out.

**Pull request merged**